index.html

```diff
@@ -14,12 +14,12 @@ <h1><a href="#0">BCIT</a></h1>
             <button class="btn-menu">Menu</button>
             <nav>
                 <ul>
-                    <li><a href="#0">Home</a></li>
-                    <li><a href="#0">Course</a></li>
-                    <li><a href="#0">Schedule</a></li>
-                    <li><a href="#0">Lessons</a></li>
-                    <li><a href="#0">Contact</a></li>
-                    <li><a href="#0">Login</a></li>
+                    <li><a href="/../index.html">Home</a></li>
+                    <li><a href="/../course.html">Course</a></li>
+                    <li><a href="/../schedule.html">Schedule</a></li>
+                    <li><a href="/../lessons.html">Lessons</a></li>
+                    <li><a href="/../contact.html">Contact</a></li>
+                    <li><a href="login.html">Login</a></li>
```

Comment:
Why use "/../name.html" instead or just "name.html"?

Reply:
You are right. I changed it.